libs/util/types/src/lib/metadata/metadata.model.ts

```diff
@@ -59,6 +59,8 @@ export interface BaseRecord {
   useLimitations: Array<string>
   licenses: Array<License>
   overviews: Array<GraphicOverview>
+  extras?: Record<string, unknown>
```

Comment:
even though I agree that we need such a field, I think we should make sure that this does not become an alternate free-form storage  in the long run; I thought that maybe having keys such as `iso19139.mySpecificfield` would make sense for example;
Not saying this means to be modified, just a thought I have

Reply:
Yes you are right, you need strictness on how we handle those properties.
I think we should mature it a bit, when we have other mappers like `dcat` and `iso19115-3 `, and see how we can handle those extras in the UI, and adapt the UI if they are not present.